app/pb_kits/playbook/pb_currency/currency.rb

```diff
@@ -61,20 +55,32 @@ def value_size
         size == "lg" ? 1 : 2
       end
 
+      def formatted_value
+        number_to_currency(configured_value)
```

Comment:
Can `value` have a default?  Does this rely on `value` being set?

Reply:
Currently, this relies on value being set. Value cannot have a default because we wouldn't want to display something that wasn't explicitly set right? At least for this kit, I think that makes sense. What do you think?

Reply:
I agree. --- I will shortly be working on a PR that adds some functionality to give us the ability to make a prop required (in the new kit definition format though). When we loop back to `currency`, we can make this a required prop.